errors_test.go

```diff
@@ -46,8 +50,10 @@ func TestHandleExitCoder_MultiErrorWithExitCoder(t *testing.T) {
 	called := false
 
 	OsExiter = func(rc int) {
-		exitCode = rc
-		called = true
+		if !called {
```

Comment:
Could you add another `ExitCoder` to this test case? I want to assert that it uses the last one (to avoid inadvertently changing that behavior).